File: src/main/java/core/listeners/StatusListener.java
package core.listeners;

import net.dv8tion.jda.api.events.user.update.UserUpdateOnlineStatusEvent;
import net.dv8tion.jda.api.hooks.ListenerAdapter;

public class StatusListener extends ListenerAdapter {

    @Override
    public void onUserUpdateOnlineStatus(UserUpdateOnlineStatusEvent event) {
	    System.out.println(event.getMember().getNickname()+" "+event.getNewOnlineStatus().toString());
    }
}

File: src/main/kotlin/core/listeners/StatusListener.kt
package core.listeners

import net.dv8tion.jda.api.events.user.update.UserUpdateOnlineStatusEvent
import net.dv8tion.jda.api.hooks.ListenerAdapter

class StatusListener : ListenerAdapter() {

    override fun onUserUpdateOnlineStatus(event: UserUpdateOnlineStatusEvent) {
        println(event.member.nickname + " " + event.newOnlineStatus.toString())
    }
}
